.yamato/config/hdrp_batchrenderergroup.metafile

```diff
@@ -60,17 +60,26 @@ platforms:
       - Linear
   - name: Linux
     apis:
-      - name: OpenGLCore
-        exclude_test_platforms:
-          - name: Standalone
-          - name: playmode_XR
       - name: Vulkan
         exclude_test_platforms:
           - name: playmode_XR
     build_configs:
       - name: mono
     color_spaces:
       - Linear
+    agents_project:
+      default:
+        type: Unity::VM::GPU
```

Comment:
Do we need a GPU here and for the standalone_build?

Reply:
This is needed for the HDRP Linux Vulkan Build job to pass. discussion on [slack](https://unity.slack.com/archives/C01AQQFTGQK/p1637072530009100?thread_ts=1636712800.000300&cid=C01AQQFTGQK)